images/alpine/cloudbuild.yaml

```diff
@@ -1,19 +1,40 @@
 steps:
-  - name: gcr.io/cloud-builders/docker
-    args:
-    - build
-    - -t
-    - gcr.io/$PROJECT_ID/alpine:$_GIT_TAG
-    - --build-arg=IMAGE_ARG=gcr.io/$PROJECT_ID/alpine:$_GIT_TAG
-    - .
+# Set up multiarch support.
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['run', '--rm', '--privileged', 'multiarch/qemu-user-static', '--reset', '-p', 'yes']
+    id: 'qemu'
+    waitFor: ['-']
```

Comment:
Looks like you are defining a set of sequential steps. This is the default behavior. Thus please drop the unnecessary waitFor items.

Reply:
'waitFor' has been removed.